tritonbench/operators/addmm/operator.py

```diff
@@ -157,6 +157,68 @@ def flops(
         flops = (2 * m * k * n) + (m * n)
         return flops
 
+    @register_metric()
+    def op_gflops(
+        self, fn_name: str, example_inputs: Any, metrics: BenchmarkOperatorMetrics
+    ) -> float:
+        """Report the raw number of GFLOPS (not GFLOPS/sec) for the addmm operation."""
+        _, mat1, mat2 = example_inputs
+        m, k = mat1.size()
+        k, n = mat2.size()
+        flops = (2 * m * k * n) + (m * n)
+        # Convert FLOPS to GFLOPS (divide by 10^9)
+        gflops = flops / 1e9
+        return gflops
+
+    @register_metric()
+    def op_gbytes(
+        self, fn_name: str, example_inputs: Any, metrics: BenchmarkOperatorMetrics
+    ) -> float:
+        """Report the raw number of gigabytes of I/O (not GB/sec) for the addmm operation."""
+        a, mat1, mat2 = example_inputs
```

Comment:
We can directly call `self.flops(fn_name, example_inputs, metrics)` to get the flops, then multiply it by `a.element_size()`.